_config.yml

```diff
@@ -68,10 +68,10 @@ defaults:
       type: "pages"
     values:
       layout: default
-  - scope:
-      type: "docs"
-    values:
-      layout: docs
+#  - scope:
+#      type: "docs"
+#    values:
+#      layout: docs
```

Comment:
Is this commented out for testing or something else?

Reply:
damn, I forgot to uncomment this 😂  fixed